iso_config.yaml

```diff
@@ -0,0 +1,37 @@
+log_file: "/var/tmp/host-os-iso-creator.log"
```

Comment:
About the "default" node: I think we should follow the same format in both yaml file, but I don't really get why we have this "default" node, so up to you.
"/var/log/host-os/iso.log" is fine.